ffi/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+#![allow(clippy::unnecessary_box_returns)] // Diplomat requires returning Boxed types
+pub mod connector;
+pub mod credssp;
+pub mod dvc;
+pub mod error;
+pub mod pdu;
+pub mod svc;
+pub mod utils;
+
+use tracing as _; // need this in the future
```

Comment:
suggestion: You can remove the dependency if it’s not used. I’m not sure we really need it in the FFI crate.